src/domain/mapDomain.ml

```diff
@@ -257,7 +257,7 @@ struct
 
   let relift = lift_f' M.relift
 end
-
+(*
```

Comment:
Is this commented out because it no longer compiles?

Reply:
It no longer compiles because the signature that the Lattice.HConsed changed such that it allows projecting away parts of the state that are not needed for the modular mode during the analysis. 
The MapDomain.HConsed is also not used anywhere.